docs/index.md

```diff
@@ -1,75 +1,75 @@
-# Table of Contents
+# 目录
```

Comment:
This file is already translated in #213, help improve [the existing one](https://github.com/rocky-linux/documentation/blob/main/docs/index.zh_CN.md) instead :-)